src/create-extension-pack.js

```diff
@@ -0,0 +1,184 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+/*
+ * Creates an extension package referencing all built-in extensions previously
+ * created during build time i.e. by executing yarn
+ * 
+ * Extensions will be skipped if a corresponding .vsix file is not found under the 'dist'
+ * folder and also not found under the extension registry.
+ */
+// @ts-check
+const fs = require('fs-extra');
+const path = require('path');
+const vsce = require('vsce');
+const yargs = require('yargs');
+
+const { computeVersion, vscodeVersion, isPublished } = require('./version');
+const { dist, extensions, theiaExtension } = require('./paths.js');
+
+const { tag, force } = yargs.option('tag', {
+    choices: ['latest', 'next']
+}).demandOption('tag')
+    .option('force', {
+        description: 'Create extension pack even if it is found to be already available',
+        boolean: true,
+        default: false
+    }).argv;
+
+const PACKAGE_JSON = 'package.json'
+const categories = ['Extension Packs'];
+const packName = 'builtin-extension-pack';
+const publisher = 'open-vsx';
+const repository = 'https://github.com/theia-ide/vscode-builtin-extensions';
```

Comment:
```suggestion
const repository = 'https://github.com/eclipse-theia/vscode-builtin-extensions';
```

Reply:
This was the result of a copy paste from the package-vsix.js script,  I have updated both accordingly !, Thanks !!